### PR TITLE
Removing the answer store

### PR DESCRIPTION
**Pull request by warrenbailey**
### What is the context of this PR?

Removes the answer store, all code should now use the new User Journey Manager.
### How to review
1. Check out this branch
2. Confirm the tests still pass
3. Complete a survey and check everything still works
- [x] Do all commits have sensible commit messages?
- [x] Is all new code unit tested?
- [x] Are there integration tests if needed?
- [x] Is there sufficient logging?
- [x] Is there documentation or is the code self-describing?
### Who worked on the PR

@warrenbailey
## Dependencies

This requires #318 to be merged first

### CODE REVIEW

**Comment by warrenbailey** [comment]
Will need rebasing once #318 is merged

**Comment by warrenbailey** [comment]
Rebased and ready for review

File: app/questionnaire/create_questionnaire_manager.py [hunk 1]
```diff
@@ -24,21 +25,22 @@ def create_questionnaire_manager():
...
 
-    # load the answer store
-    answer_store = factory.create("answer-store")
-
     # load the validation store
     validation_store = factory.create("validation-store")
 
+    user_journey_manager = UserJourneyManager.get_instance()
+    if not user_journey_manager:
+        logger.error("Constructing brand new User Journey Manager")
```
Comment (LJBabbage) [inline app/questionnaire/create_questionnaire_manager.py]: Should be info not error

File: app/questionnaire_state/user_journey_manager.py [hunk 2]
```diff
@@ -150,6 +150,28 @@ def _get_first_location(self):
...
+        answers = []
+        while page:
+            page_answers = page.page_state.get_answers()
+            answers.extend(page_answers)
+            page = page.next_page
+
+        for answer in answers:
+            id = answer.id
+            value = answer.input
+            answers_dict[id] = value
```
Comment (LJBabbage) [inline app/questionnaire_state/user_journey_manager.py]: I prefer
answers_dict[answer.id] = answer.input

**Comment by LJBabbage** [comment]
Looks good bar a couple of minor personal preferences

File: app/questionnaire/create_questionnaire_manager.py (hunk 1 above)
Comment (warrenbailey) [inline app/questionnaire/create_questionnaire_manager.py]: done

File: app/questionnaire_state/user_journey_manager.py (hunk 2 above)
Comment (warrenbailey) [inline app/questionnaire_state/user_journey_manager.py]: done

**Comment by dhilton** [comment]
Can we squash that last commit or amend it's text? Makes for a cleaner commit history.

**Pull request merged**